openquake/calculators/scenario_damage.py

```diff
@@ -213,6 +212,14 @@ def post_execute(self, result):
         # damage by event
         eid_dmg_dt = self.crmodel.eid_dmg_dt()
         d_event = numpy.array(sorted(result['d_event'].items()), eid_dmg_dt)
+        # sanity check on the total number of assets, same for each event
+        num_sums = len(set(d['dmg'].sum() for d in d_event))
+        if self.param['approx_ddd'] and num_sums > 1:
+            logging.warning('Due to numeric errors the sum of assets in each '
```

Comment:
"Due to rounding errors, the total number of structural units may not exactly identical for each event in the dmg_by_event table."